src/plugin.js

```diff
@@ -2,6 +2,10 @@ import videojs from 'video.js';
 import window from 'global/window';
 import document from 'global/document';
 
+// The logic below is used to check if flash is disabled or not installed
+// in IE browser and show the appropriate message to the user.
+const isFlashSupported = videojs.getComponent('Flash').isSupported();
```

Comment:
you may just want to call this function when you need it so that this code will be easier to test, rather than making it a const at the top